torch/tensor.py

```diff
@@ -378,6 +378,8 @@ def __rfloordiv__(self, other):
     __ge__ = _C._TensorBase.ge
     __abs__ = _C._TensorBase.abs
 
+    __hash__ = None
```

Comment:
It might be a bit more friendly to suggest using `id` if you want to put tensors in a dict instead of simply saying "you can't hash this".

Reply:
Would something like this do:
```
def __hash__(self):
    raise TypeError("Tensors are not hashable. If you would like to put tensors in a set or a dict, consider using id")
```